### DISCUSSION TITLE
Roadmap plan

### DISCUSSION THREAD

**Discussion by SilvioGrosso** [RawLabo/QuickRawPicker#24] [General]
Hello @qdwang 

Just out of sheer curiosity... :-)

Do you have some basic Roadmap planned for the future?
In essence, a list of features you want to develop?

**Comment by qdwang** [comment]
Currently I do not have any roadmap plan. 

I'd like to complete the very first stable version and  submit some posts on more websites like dpreivew forum etc.

Do you have any recommends?

**Comment by SilvioGrosso** [comment]
Hello @qdwang 

> Do you have any recommends?

Nope :-)
Simply because QuickRawPicker is already quite powerful even with its "minimalist" approach.
There are already many open source softwares with far more tools (e.g. Digikam).

Thanks a lot for releasing your software :-)
